catalog/pipeline/fbc-release/0.9/README.md

```diff
@@ -0,0 +1,27 @@
+# FBC Release Pipeline
+
+FBC Release Pipeline
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the EnterpriseContractPolicy | No | - |
+| fromIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| targetIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| overWriteFromIndex | Boolean indicating if the fromIndex should be overwritten | Yes | "false" |
+| binaryImage | OCP binary image to be baked into the index image | Yes | "" |
+| buildTags | List of additional tags the internal index image copy should be tagged with | Yes | "[]" |
+| addArches | List of arches the index image should be built for | Yes | "[]" |
+| requester | Name of the user that requested the signing, for auditing purposes | No | - |
+| signingConfigMapName | The ConfigMap Name required by the Pipeline | Yes | "hacbs-signing-pipeline-config" |
+| fbcPublishingCredentials | Secret used to publish the built index image  | Yes | "fbc-publishing-pull-secret" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | - |
+| buildTimeoutSeconds | Max seconds to wait until the build finishes | Yes | - |
+
+## Changelog
+
+## Changes since 0.4
```

Comment:
This changelog looks a little messed up. v0.9 has the latest change "since 0.4" and line 26 has no T in the first word. Maybe copy paste issue